src/main/java/org/opensearch/geospatial/ip2geo/action/GetDatasourceResponse.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.geospatial.ip2geo.action;
+
+import java.io.IOException;
+import java.time.Instant;
+import java.util.List;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.extern.log4j.Log4j2;
+
+import org.opensearch.action.ActionResponse;
+import org.opensearch.common.io.stream.StreamInput;
+import org.opensearch.common.io.stream.StreamOutput;
+import org.opensearch.core.ParseField;
+import org.opensearch.core.xcontent.ToXContentObject;
+import org.opensearch.core.xcontent.XContentBuilder;
+import org.opensearch.geospatial.ip2geo.jobscheduler.Datasource;
+
+/**
+ * Ip2Geo datasource get request
+ */
+@Getter
+@Setter
+@Log4j2
```

Comment:
remove log4j2 from here too. Please make sure that you check all your classes and remove the unsed annotations.

Reply:
Done